lib/commands/context.js

```diff
@@ -333,7 +339,16 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
   if (opts.pageLoadStrategy) {
     caps.pageLoadStrategy = opts.pageLoadStrategy;
   }
+  if (caps.chromeOptions.androidPackage === 'chrome') {
```

Comment:
Is it safer to compare as case insensitive?

Reply:
yeah, i was not sure where `WEBVIEW_chrome` comes from and if we can rely on it always being the same. I was assuming we could, but maybe it's a bad assumption.

Reply:
decided to make it case insensitive